client/src/pages/DuelOverview.jsx

```diff
@@ -99,10 +100,9 @@ const DuelOverview = ({
 
   const { currentUserScore, opponentScore, opponent, inProgress, finishedDate, TTL } = data;
 
-  const remainingTimeBeforeRemove = (function () {
-    if (!finishedDate) return TTL;
-    const timeDiff = new Date().getTime() - finishedDate;
-    return TTL - Math.floor(timeDiff / (1000 * 60 * 60 * 24));
+  const removingTime = (function () {
+    const d = incrementDate({ days: Number(TTL) }, new Date(finishedDate));
+    return setDateToNextHour(d, 0, 1);
```

Comment:
```suggestion
    const date = incrementDate({ days: Number(TTL) }, new Date(finishedDate));
    return setDateToNextHour(date, 0, 1);
```
Because one-letter variable names are a bit too much ... Jean-François Couchot :)